tests/python/contrib/test_arm_compute_lib/infrastructure.py

```diff
@@ -42,27 +50,15 @@ def __init__(self):
     @classmethod
     def _get_remote(cls):
         """Get a remote (or local) device to use for testing."""
-        if cls.use_remote:
-            # Here you may adjust settings to run the ACL unit tests via a remote
-            # device using the RPC mechanism. Use this in the case you want to compile
-            # an ACL module on a different machine to what you run the module on i.e.
-            # x86 -> AArch64.
-            #
-            # Use the following to connect directly to a remote device:
-            # device = rpc.connect(
-            #     hostname="0.0.0.0",
-            #     port=9090)
-            #
-            # Or connect via a tracker:
-            # device = tvm.autotvm.measure.request_remote(
-            #     host="0.0.0.0",
-            #     port=9090,
-            #     device_key="device_key",
-            #     timeout=1000)
-            #
-            # return device
-            raise NotImplementedError(
-                "Please adjust these settings to connect to your remote device.")
+        if cls.rpc_host != "localhost":
+            if cls.rpc_device_key:
+                device = request_remote(cls.rpc_device_key,
+                                        cls.rpc_host,
+                                        cls.rpc_port,
+                                        timeout=1000)
+            else:
+                device = rpc.connect(cls.rpc_host, cls.rpc_port)
+            return device
```

Comment:
nitpick: You can simplify this function by having only one return statement. I suggest taking this return out of the `if` (also the one on line 64)